### PR TITLE
Start to update doc

### PR DESCRIPTION
**Pull request by jeremybarbet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
- - Gradle 4
- - Unit and integration tests
- - Continuous delivery support for Travis and Bitrise
+- Opinionated dependencies
+- React Native Navigation 2
+- TypeScript 3
+- Cocoapods
+- Gradle 4
+- Unit and integration tests
+- Continuous delivery support for Travis and Bitrise
```
Comment (birkir) [inline README.md]: Maybe "Continuous delivery support for Microsoft App Center"

File: docs/DEBUGGING_IOS.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+## iOS
+
+You gonna have to build a `main.jsbundle` file to add into Xcode and provide to the app, use the following command:
```
Comment (birkir) [inline docs/DEBUGGING_IOS.md]: Are we sure this is absolutely correct? I am able to just change to "Release" scheme and it works fine.

Is this maybe an XCode 9 only issue?

File: docs/FEATURES.md [hunk 3]
```diff
@@ -16,65 +10,72 @@ These dependencies are shipped by default. They are strictly opinionated by Ueno
...
+### typescript
 
-### date-fns
+We like Typescript. It takes a bit of time to get used to it, but it makes the app more solid, easier to read, it's very well embed with vscode for example, and it makes the build time faster.
 
-We like date-fns better than momentjs because it is modular.
+### mobx-state-tree
 
-### core-decorators
+We use mobx-state-tree to manage our state. It's awesome. It's the new version, it's more complicated, but it's awesome.
```
Comment (birkir) [inline docs/FEATURES.md]: 🤣😅

File: docs/DEBUGGING_IOS.md (hunk 2 above)
Comment (jeremybarbet) [reply docs/DEBUGGING_IOS.md]: Alright, I run it and it works. But I updated to Xcode 10, so I can't tell if I did something wrong or not before. I'll change it with the "release" switch behavior, and keep the rest if people are still having troubles

**Pull request merged**